### PR TITLE
Backend include roll api

### PR DESCRIPTION
**Pull request by swetak20**
API https://search.pclub.in/api/student?username=swetak20 modified to work for https://search.pclub.in/api/student?roll=201037 too.

### CODE REVIEW

File: backend/server.py [hunk 1]
```diff
@@ -52,12 +52,17 @@ def get_all_students():
 @app.route('/student')
 def get_particular_student():
     username = request.args.get('username')
-    if username is None or username is '':
+    roll = request.args.get('roll')
+    if ((username == None or username == '' )and (roll == None or roll == '')) :
```
Comment (bhuvansingla) [inline backend/server.py:56]: - Use `is` instead of `==` to maintain consistency in the codebase
- Add space before `and`
- Remove space before `:`

**Comment by swetak20** [comment]
The virtual env required the bin folder to be created, but removed the folder so that it does not disturb the setup, let me know if this works fine?

**Comment by dryairship** [comment]
lgtm

**Review by dryairship — APPROVED** [review]
lgtm

**Pull request merged**